src/lib/isTimezone.js

```diff
@@ -0,0 +1,601 @@
+  'asia/atyrau',
+  'asia/baghdad',
+  'asia/bahrain',
+  'asia/baku',
+  'asia/bangkok',
+  'asia/barnaul',
+  'asia/beirut',
+  'asia/bishkek',
+  'asia/brunei',
+  'asia/calcutta',
+  'asia/chita',
+  'asia/choibalsan',
+  'asia/chongqing',
+  'asia/chungking',
+  'asia/colombo',
+  'asia/dacca',
+  'asia/damascus',
+  'asia/dhaka',
+  'asia/dili',
+  'asia/dubai',
+  'asia/dushanbe',
+  'asia/famagusta',
+  'asia/gaza',
+  'asia/harbin',
+  'asia/hebron',
+  'asia/ho_chi_minh',
+  'asia/hong_kong',
+  'asia/hovd',
+  'asia/irkutsk',
+  'asia/istanbul',
+  'asia/jakarta',
+  'asia/jayapura',
+  'asia/jerusalem',
+  'asia/kabul',
+  'asia/kamchatka',
+  'asia/karachi',
+  'asia/kashgar',
+  'asia/kathmandu',
+  'asia/katmandu',
+  'asia/khandyga',
+  'asia/kolkata',
+  'asia/krasnoyarsk',
+  'asia/kuala_lumpur',
+  'asia/kuching',
+  'asia/kuwait',
+  'asia/macao',
+  'asia/macau',
+  'asia/magadan',
+  'asia/makassar',
+  'asia/manila',
+  'asia/muscat',
+  'asia/novokuznetsk',
+  'asia/novosibirsk',
+  'asia/omsk',
+  'asia/oral',
+  'asia/phnom_penh',
+  'asia/pontianak',
+  'asia/pyongyang',
+  'asia/qatar',
+  'asia/qyzylorda',
+  'asia/rangoon',
+  'asia/riyadh',
+  'asia/saigon',
+  'asia/sakhalin',
+  'asia/samarkand',
+  'asia/seoul',
+  'asia/shanghai',
+  'asia/singapore',
+  'asia/srednekolymsk',
+  'asia/taipei',
+  'asia/tashkent',
+  'asia/tbilisi',
+  'asia/tehran',
+  'asia/tel_aviv',
+  'asia/thimbu',
+  'asia/thimphu',
+  'asia/tokyo',
+  'asia/tomsk',
+  'asia/ujung_pandang',
+  'asia/ulaanbaatar',
+  'asia/ulan_bator',
+  'asia/urumqi',
+  'asia/ust-nera',
+  'asia/vientiane',
+  'asia/vladivostok',
+  'asia/yakutsk',
+  'asia/yangon',
+  'asia/yekaterinburg',
+  'asia/yerevan',
+  'atlantic/azores',
+  'atlantic/bermuda',
+  'atlantic/canary',
+  'atlantic/cape_verde',
+  'atlantic/faeroe',
+  'atlantic/faroe',
+  'atlantic/jan_mayen',
+  'atlantic/madeira',
+  'atlantic/reykjavik',
+  'atlantic/south_georgia',
+  'atlantic/st_helena',
+  'atlantic/stanley',
+  'australia/act',
+  'australia/adelaide',
+  'australia/brisbane',
+  'australia/broken_hill',
+  'australia/canberra',
+  'australia/currie',
+  'australia/darwin',
+  'australia/eucla',
+  'australia/hobart',
+  'australia/lhi',
+  'australia/lindeman',
+  'australia/lord_howe',
+  'australia/melbourne',
+  'australia/north',
+  'australia/nsw',
+  'australia/perth',
+  'australia/queensland',
+  'australia/south',
+  'australia/sydney',
+  'australia/tasmania',
+  'australia/victoria',
+  'australia/west',
+  'australia/yancowinna',
+  'brazil/acre',
+  'brazil/denoronha',
+  'brazil/east',
+  'brazil/west',
+  'canada/atlantic',
+  'canada/central',
+  'canada/eastern',
+  'canada/mountain',
+  'canada/newfoundland',
+  'canada/pacific',
+  'canada/saskatchewan',
+  'canada/yukon',
+  'cet',
+  'chile/continental',
+  'chile/easterisland',
+  'cst6cdt',
+  'cuba',
+  'eet',
+  'egypt',
+  'eire',
+  'est',
+  'est5edt',
+  'etc/gmt',
+  'etc/gmt+0',
```

Comment:
These with patterns could easily be regex instead?

Reply:
What is the purpose? Speed, maintainability? Why this is bad solution?